jupyter_nbformat/__init__.py

```diff
@@ -40,7 +40,17 @@ class NBFormatError(ValueError):
     pass
 
 # no-conversion singleton
-NO_CONVERT = object()
+class Sentinel(object):
+    """Prevent nbformat to convert notebooks to most recent version.
```

Comment:
The docstring is really for the instance, not the class. Maybe it should have a second argument to take a description?